tests/kafkatest/services/kafka/kafka.py

```diff
@@ -303,6 +308,22 @@ def leader(self, topic, partition=0):
         self.logger.info("Leader for topic %s and partition %d is now: %d" % (topic, partition, leader_idx))
         return self.get_node(leader_idx)
 
+    def controller(self):
+        """ Get the controller ID
```

Comment:
Outdated comment? I think this returns the controller node, not the controller ID